Source/Fuse.Controls.TimePicker/TimePicker.Docs.uno

```diff
@@ -0,0 +1,67 @@
+namespace Fuse.Controls
+{
+	/**
+		Displays a component to select a time.
+
+		Currently, the TimePicker only has native implementations, so it should be contained in a @NativeViewHost.
+
+		A `TimePicker` can be used to select a specific time value. The type of its `Value` property is `Uno.DateTime`,
+		which is marshalled automatically to and from the JavaScript `Date` type. This makes interaction between JavaScript
+		and the `TimePicker` type seamless via databinding. If you plan to wrap a `TimePicker` in a UX component and use a
+		UX property to hook up this value, the `Uno.DateTime` type should be used.
+
+		Both `Uno.DateTime` and JS' `Date` type represent a specific timestamp. These types have both date and time
+		components, and their interpretation depends on a given time zone, which can cause a great deal of confusion. To
+		simplify all of this and ensure consistent behavior accross different time zones and locales, `TimePicker` will assume
+		incoming values are relative to UTC, and truncate the date component to the Unix epoch (1 Jan 1970), effectively
+		ignoring the date component altogether. Similarly, values read from `TimePicker` properties will only consist of a time
+		component at on 1 Jan 1970. This makes values going to/from the `TimePicker` control easy to create and interpret
+		consistently, but also means that if a value with a date component other than the unix epoch is written to TimePicker`'s
+		`Value` property, subsequent values read from the property may not match the written value, as the date component will
+		have been truncated.
+
+		Note that this control should not be used to deal with historic dates, as calendar/date and timestamp translation is
```

Comment:
I thought this just selected time, not date?

Reply:
It only selects time, however it does so by using `DateTime` values, which inherently have date components as well. When implementing the date pickers this was a big problem, as different locales would have completely different interpretations of dates from a given timestamp due to when different calendars were adopted.

This may not actually be a problem with the time pickers specifically, but I figured we could be conservative here and copy this comment over as the same problems could affect the interpretation of time values due to DST or similar. But perhaps that's misguided; it's possible I could test more to see if it's a problem, although it's a bit difficult to test each possible scenario without buying some expensive plane tickets :)